examples/cpp/shared_recording/main.cpp

```diff
@@ -0,0 +1,22 @@
+#include <unistd.h>
+#include <iostream>
+#include <sstream>
+
+#include <rerun.hpp>
+#include <rerun/demo_utils.hpp>
+
+using rerun::demo::grid3d;
+
+int main() {
+    const auto rec =
+        rerun::RecordingStream("rerun_example_shared_recording", "my_shared_recording");
+    rec.spawn().exit_on_failure();
+
+    int pid = getpid();
+    std::ostringstream oss;
+    oss << "Hello from " << pid;
+
+    rec.log("updates", rerun::TextLog(oss.str()));
```

Comment:
stringstream works, but you could do this simpler by just going
```cpp
std::string("Hello from ") + std::to_string(pid)
```

Reply:
I mean, just look at this beauty :heart_eyes: 
```c++
    rec.log("updates", rerun::TextLog(std::string("Hello from ") + std::to_string(getpid())));
```